codespell_lib/data/dictionary.txt

```diff
@@ -36968,6 +37001,7 @@ vas->was
 vasall->vassal
 vasalls->vassals
 vaue->value
+vaues->values, vagues,
```

Comment:
`vagues`? Isn't `vague` an adjective only?

Reply:
It's a bit weird, and I guess archaic: [Wiktionary](https://en.wiktionary.org/wiki/vague#Verb), [Websters 1913](https://www.websters1913.com/words/Vague). But [all contemporary references](https://www.onelook.com/?w=vague&ls=a) seem to have it only as an adjective. So I guess we can skip it; now done.

Reply:
Also added `vaue`, `vaued`.